cookbooks/aws-parallelcluster-slurm/recipes/config_head_node.rb

```diff
@@ -86,6 +86,13 @@
   mode '0644'
 end
 
+template "#{node['cluster']['slurm']['install_dir']}/etc/slurmdbd.conf" do
+  source 'slurm/slurmdbd.conf.erb'
+  owner "#{node['cluster']['slurm']['dbduser']}"
+  group "#{node['cluster']['slurm']['dbdgroup']}"
+  mode '0600'
+end
+
```

Comment:
[Question] If I'm reading this correctly, we are creating slurmdb configuration files, whether or not slurm accounting is enabled?

Reply:
Yes, but only with generic parameters - with this configuration slurmdbd wouldn't be working. And we don't enable the slurmdbd service.

Reply:
Can this be problematic in case the user is already enabling accounting by themselves and want to continue to do so?Wouldn't it be better to not produce any file in case the user does not select accounting settings so that we are less breaking with respect to existing solutions?

Reply:
This happens only at cluster creation time. The customers are free to manipulate the base `slurmdbd.conf`and they are good to go.

Generating the whole `slurmdbd.conf` in a dynamic manner would be inconsistent with respect to how we generate the other Slurm configuration files, and would also mean that at cluster update time we would erase any customer customization for the slurmdbd that is not generated via `pcluster_slurm_config_generator.py`.

I'll think about this, but up to now all the alternatives that come to my mind are either worse, or require that we re-think the way users can customize the Slurm configuration.

Reply:
The idea is:
- at create time, write file only if our accounting is enabled;
- at update time when feature gets enabled, overwrite any possible slurmdbd config files.

Reply:
Done.